recipe/meta.yaml

```diff
@@ -26,6 +26,26 @@ test:
     - python
   files:
     - test.c
+  commands:
+    - test -f $LIBRARY_INC\\mpir.h              # [win]
+    - test -f $LIBRARY_INC\\mpirxx.h            # [win]
+    - test -f $LIBRARY_INC\\gmp.h               # [win]
+    - test -f $LIBRARY_INC\\gmpxx.h             # [win]
+    - test -f $LIBRARY_LIB\\mpir.lib            # [win]
+    - test -f $LIBRARY_LIB\\mpirxx.lib          # [win]
+    - test -f $LIBRARY_LIB\\gmp.lib             # [win]
+    - test -f $LIBRARY_LIB\\gmpxx.lib           # [win]
```

Comment:
So, `test -f <FILE>` doesn't work on Windows. Instead try `if not exist <FILE> exit 1`.